src/shared/getApiMetrics.ts

```diff
@@ -63,6 +68,21 @@ export function getApiMetrics(messages: ClineMessage[]) {
 			}
 		} else if (message.type === "say" && message.say === "condense_context") {
 			result.totalCost += message.contextCondense?.cost ?? 0
+		} else {
+			// kilocode_change start
+			if (message.type === "ask" && message.ask === "tool" && message.text) {
+				const fastApplyResult = safeJsonParse<ClineSayTool>(message.text)?.fastApplyResult
+				if (fastApplyResult?.tokensIn) {
+					result.totalTokensIn += fastApplyResult.tokensIn
+				}
+				if (fastApplyResult?.tokensOut) {
+					result.totalTokensIn += fastApplyResult.tokensOut
+				}
+				if (fastApplyResult?.cost) {
+					result.totalTokensIn += fastApplyResult.cost
+				}
```

Comment:
Should these be `totalTokensIn/totalTokensOut/totalCost`?
```suggestion
				result.totalTokensIn += fastApplyResult?.tokensIn ?? 0
				result.totalTokensOut += fastApplyResult.tokensOut ?? 0
				result.totalCost += fastApplyResult.cost ?? 0
```